client-mobile/src/containers/ShoppingListContainer/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { IconName } from "@client/components/atoms/Icon/@types"
+import { Typography } from "@client/components/atoms/Typography"
+import { TypographyVariant } from "@client/components/atoms/Typography/@types"
+import { Button } from "@client/components/molecules/Button"
+import { EmptyResult } from "@client/components/molecules/EmptyResult"
+import { shoppingListContainerDefaultStyles } from "@client/containers/ShoppingListContainer/styles"
+import { View } from "dripsy"
+import { useTranslation } from "react-i18next"
+
+export const ShoppingListContainer: React.FC = () => {
+  const { t } = useTranslation()
+
+  const { mainWrapper, textWrapper, button } = shoppingListContainerDefaultStyles
+
+  return (
+    <View sx={mainWrapper}>
+      <View sx={textWrapper}>
+        <Typography variant={TypographyVariant.Title}>{t("Shopping lists")}</Typography>
```

Comment:
```suggestion
        <Typography variant={TypographyVariant.Title}>{Shopping lists}</Typography>
```
`t()` function should be a part of `Typography` component.